internal/proto/query_coord.proto

```diff
@@ -223,7 +223,7 @@ message SegmentLoadInfo {
   int64 collectionID = 3;
   int64 dbID = 4;
   int64 flush_time = 5;
-  repeated data.FieldBinlog binlog_paths = 6;
+  data.SegmentBinlogs segment_binlogs = 6;
```

Comment:
Dangerous modification!
 Consider whether it will cause the problem of not being able to load old data！

Reply:
fixed